### ISSUE TITLE
How to install PKGs in /pkg_example/ folder

### ISSUE THREAD

**Issue by yefeiyu** [HappyZ/dpt-tools#63]
The wiki article "PKGs I Made" hasn't detailed steps.

**Comment by yefeiyu** [comment]
I have rooted my dpt-rp1 on Jan-10-2019, and now I wanna to use bluetooth keyboard for type.
I found that you update the new folder of `pkg_example` 8 days ago, but I don't know how to install and use them.
Is it possible to write a detailed description?

**Comment by HappyZ** [comment]
just use `fw`

**Comment by yefeiyu** [comment]
```
>>> fw
[info] Please make sure you have charged your battery before this action.
>>> Please enter the pkg file path: 
```
I don't know how should I input the `file path`, I am use MacOS, and I git clone entire `dpt-tools` down. When I input the command `python3 dpt-tools.py -ip xxx.xxx` I posited /home/xxx/dpt-tools.

**Comment by HappyZ** [comment]
closed as a new issue has been created #65
